lib/run.go

```diff
@@ -118,6 +118,14 @@ func run(c *cli.Context, name string, gapiObj gapi.GAPI) error {
 		obj.PgpIdentity = &us
 	}
 
+	if cliContext.Bool("no-network-standalone") {
```

Comment:
In general we try and put this "defaults" logic inside of the etcd/ if possible, as lib/ just copies the values in... Do you think that's possible? Look at the `const` default, for example. HTH

Reply:
I'll look into that.

Reply:
Great!

Reply:
This has been moved into `etcd/etcd.go`